docs/rules.md

```diff
@@ -58,6 +58,7 @@
 * [no-whitespace-for-layout](rule/no-whitespace-for-layout.md)
 * [no-whitespace-within-word](rule/no-whitespace-within-word.md)
 * [no-yield-only](rule/no-yield-only.md)
+* [no-yield-block-params-to-else-inverse](rule/no-yield-block-params-to-else-inverse.md)
```

Comment:
FYI, this file has been deleted in the latest master, and you'll want to run `yarn update` to update our README instead.